frontend/packages/core/src/Input/form.tsx

```diff
@@ -9,6 +9,7 @@ const Form = styled.form({
 
 const FormRow = styled.div({
```

Comment:
the form rows get misaligned when you type into them
![Screen Shot 2022-07-26 at 11 59 12 AM](https://user-images.githubusercontent.com/39421794/181054070-a663551c-6451-4530-8c51-b31fd50e1570.png)

Reply:
oh actually i think it's due to the new styling changes to the text field

Reply:
Hmm, didn't see this one, I'll take a look. Which workflow was this?

Reply:
the k8s dashboard

Reply:
Alright the secondary issue should be fixed now, I need to see if theres a better way to do this however because I'm currently not a fan.

Reply:
Alright, was able to modify it and removed some extra code, looks like we should be good now.